### ISSUE TITLE
Update AJAX Redux actions to expect only the API suffix

### ISSUE THREAD

**Issue by mjoyce91** [18F/State-TalentMAP#248]
Do not pass API URL to components and then back to Redux actions - simply pass the suffix (ex: `/position/345`) and reference the API prefix externally.

**Comment by jseppi** [comment]
Or just the relevant identifier (`345` in this case). The fact that it uses a URL can be completely encapsulated.

**Comment by mjoyce91** [comment]
Good point - `/position/` doesn't need to live in the component either.

**Comment by BrownCA5** [comment]
Need associated milestone added.
